keanu-project/src/main/java/io/improbable/keanu/tensor/intgr/ScalarIntegerTensor.java

```diff
@@ -519,9 +519,7 @@ public BooleanTensor elementwiseEquals(Integer value) {
 
     @Override
     public String toString() {
-        return "ScalarIntegerTensor{" +
-            "value=" + value +
-            '}';
+        return String.format("[[%d]]", value);
```

Comment:
The number of [s will depend on the rank of the scalar. 1x1x1 would be [[[value]]].